bin/prep-release

```diff
@@ -8,40 +8,37 @@
 set -e
 
 if [ -z "$1" ]; then
-  echo "usage: bin/prep-release VERSION" >&2
+  printf "usage: bin/prep-release VERSION\n" >&2
```

Comment:
FWIW my own style is to use simple `echo` for simple strings (as all these are), and only go to the clunkier `printf` when needed (for escapes, variables, etc).